server/lists/index.js

```diff
@@ -164,6 +176,12 @@ module.exports = {
             type: listService.types.STATIC,
             adapter: entityListItemsAdapter
         },
+        MPAM_CONTRIBUTION_BUSINESS_AREAS: {
+            client: 'INFO',
+            endpoint: '/entity/list/MPAM_CONTRIBUTION_BUSINESS_AREAS',
+            type: listService.types.STATIC,
+            adapter: entityListItemsAdapter
+        },
```

Comment:
Just checking - this was brought over in the merge from main?

Reply:
yeh that was my work on main to genericise.